### PR TITLE
Fix issue #691

### LINKED ISSUE #691: DOC: Update or remove broken links to QE lectures
For example: https://github.com/QuantEcon/QuantEcon.py/blob/15aa789198f28969c2e7aea005720168a244188d/quantecon/_lss.py#L4-L7

### PR DESCRIPTION
**Pull request by HengchengZhang**
Closes    #691 

Removed the following broken links

https://github.com/QuantEcon/QuantEcon.py/blob/8a61df30318a4b2badbff17744782762ee1bac1f/quantecon/_inequality.py#L107-L111

https://github.com/QuantEcon/QuantEcon.py/blob/8a61df30318a4b2badbff17744782762ee1bac1f/quantecon/_kalman.py#L4-L7

https://github.com/QuantEcon/QuantEcon.py/blob/8a61df30318a4b2badbff17744782762ee1bac1f/quantecon/_kalman.py#L57-L60

https://github.com/QuantEcon/QuantEcon.py/blob/8a61df30318a4b2badbff17744782762ee1bac1f/quantecon/bld.bat#L6-L8

https://github.com/QuantEcon/QuantEcon.py/blob/8a61df30318a4b2badbff17744782762ee1bac1f/quantecon/_lae.py#L12-L15

https://github.com/QuantEcon/QuantEcon.py/blob/8a61df30318a4b2badbff17744782762ee1bac1f/quantecon/meta.yaml#L64-L66

### CODE REVIEW

**Comment by oyamad** [comment]
@HengchengZhang Thank you for the PR!

* I think the links to lectures should be updated to new ones, rather than to remove.
* Benhabib, Bisin, Luo, 2017 seems to have been published in AER. Can anyone confirm?
* @mmcky What are the files `bld.bat`, `build.sh` and `meta.yaml` for?

**Comment by shlff** [comment]
> @HengchengZhang Thank you for the PR!
> 
> * I think the links to lectures should be updated to new ones, rather than to remove.
> * Benhabib, Bisin, Luo, 2017 seems to have been published in AER. Can anyone confirm?
> * @mmcky What are the files `bld.bat`, `build.sh` and `meta.yaml` for?

Thanks @oyamad and @HengchengZhang . I can confirm the publication of Benhabib, Bisin, and Luo in AER:
- https://www.aeaweb.org/articles?id=10.1257/aer.20151684

**Comment by HengchengZhang** [comment]
Thanks for your advice and confirmation @oyamad  and @shlff.

I've updated the correct links to the files.

I think this domain is no longer being used
https://github.com/QuantEcon/QuantEcon.py/blob/eeb0865908da0a7f528ad73e99028b225b8f26d4/quantecon/meta.yaml#L62

Shall we change it to `quantecon.org`?

**Comment by mmcky** [comment]
thanks @HengchengZhang 

> Shall we change it to quantecon.org?

Yes please.

**Comment by mmcky** [comment]
> @mmcky What are the files bld.bat, build.sh and meta.yaml for?

I think we can remove these. 

They were initially there to support `conda-forge` releases but that infrastructure is probably old and `quantecon` has been making its way into default `anaconda` channels. 

https://repo.anaconda.com/pkgs/main/linux-64/

Although I don't see `0.6.0` which is surprising given it's been out since `December 2022`

**Comment by HengchengZhang** [comment]
> thanks @HengchengZhang
> 
> > Shall we change it to quantecon.org?
> 
> Yes please.

Thanks @mmcky, updated.

**Review by mmcky — APPROVED** [review]
This looks good to me. Thanks for updating the links to the new links @HengchengZhang. Thanks @oyamad for the review.

File: quantecon/_inequality.py [hunk 1]
```diff
@@ -108,7 +108,8 @@ def shorrocks_index(A):
     ----------
     .. [1] Wealth distribution and social mobility in the US:
        A quantitative approach (Benhabib, Bisin, Luo, 2017).
-       https://www.econ.nyu.edu/user/bisina/RevisionAugust.pdf
+       https://www.aeaweb.org/articles?id=10.1257/aer.20151684
```
Comment (oyamad) [inline quantecon/_inequality.py:111]: Update "Benhabib, Bisin, Luo, 2017" as well? (to publication information in AER)

File: quantecon/_kalman.py [hunk 2]
```diff
@@ -1,10 +1,10 @@
 """
 Implements the Kalman filter for a linear Gaussian state space model.
 
-References
-----------
+References 
+----------
```
Comment (oyamad) [inline quantecon/_kalman.py:5]: Please remove trailing whitespace (at some other places as well).

**Comment by HengchengZhang** [comment]
Many thanks for your review @oyamad, I've applied the changes.

**Pull request merged**